docs/new-guides.md

```diff
@@ -0,0 +1,23 @@
+# Proposing a new guide
+
+Thanks for your interest in suggesting a new Open Source Guide! Your suggestions help us decide what to work on next.
+
+## If you'd like to propose a new guide:
+
+1. **[Open an issue](https://github.com/github/opensource.guide/issues)** with your suggestion. Please explain:
+  * Why the topic is broadly useful to open source contributors
+  * Why it doesn't fit into an existing guide
+  * 3-5 bullets of expected sub-topics
+2. We (likely @bkeepers or @nayafia) will label the issue ```new-guide``` and share feedback.
+
+At this point, we may decide the topic doesn't warrant a new guide and close the issue, or we may recommend adding the information to an existing guide. That's okay! All suggestions are welcomed contributions, because they help us understand what our community is thinking.
+
+## If we decide to move forward on a new guide:
+
+1. We'll add a ```research``` label to your issue.
+2. We'll start using your issue to collect existing community resources (links, blog posts, projects). Everybody is welcome to contribute.
```

Comment:
May be premature optimization, but it I'd love to detail what a generic process for this looks like. e.g.:

- What else has been written on this topic?
- What projects do this well?
- Who might be people with valuable experience that we should talk to?